branch-diff.js

```diff
@@ -105,12 +105,16 @@ function diffCollected (options, branchCommits, callback) {
 }
 
 
-function printCommits (list, format) {
+function printCommits (list, format, reverse) {
   if (format === 'sha') {
     list = list.map((commit) => `${commit.sha.substr(0, 10)}`)
   } else {
     list = list.map((commit) => commitToOutput(commit, format === 'simple', ghId))
   }
+  
+  if (reverse) {
```

Comment:
remove braces around this, they should be left out for one-liners like this, kept in (like just above) when there's an `else` or the statement gets complicated and spans >1 line even when it's technically a one-liner.